addons/apecloud-mysql/scripts/etcd-post-start.sh

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
-endpoints=${ETCD_SERVER:-'127.0.0.1:2379'}
+if [ -n "$LOCAL_ETCD_POD_LIST" ]; then
+  IFS=',' read -ra ETCD_POD_ARRAY <<< "$LOCAL_ETCD_POD_LIST"
+  endpoints=""
+  for pod in "${ETCD_POD_ARRAY[@]}"; do
+    endpoints+="${pod}.${LOCAL_ETCD_HEADLESS}.${KB_NAMESPACE}.svc.cluster.local:${LOCAL_ETCD_PORT},"
```

Comment:
why not use LOCAL_ETCD_HEADLESS?

Reply:
and you may have a try with vars expression

Reply:
Isn't it true that I am currently using LOCAL_ETCD_HEADLESS, meaning directly using the HEADLESS service?

Reply:
> Isn't it true that I am currently using LOCAL_ETCD_HEADLESS, meaning directly using the HEADLESS service?

yeah, you're correct, the pod's headless svc should be pod name ++ Local_ETC_HEADLESSS,